### DISCUSSION TITLE
Invalid data.zip version 2024-02-27 05:32

### DISCUSSION THREAD

**Discussion by thetengda** [groops-devs/groops#244] [Q&A]
Dear developers,
The data.zip (2024-02-27 05:32) on your FTP server seems not to have been uploaded correctly, I have tried to download and unzip the file several times and the error alert seems to indicate that the file is incomplete. Is there a way to download an older version please?

**Please refer to the following information:**
huang@DESKTOP-EV8L72E:/mnt/d/dev/groops/data$ unzip data.zip
Archive:  data.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
note:  data.zip may be a plain executable, not an archive
unzip:  cannot find zipfile directory in one of data.zip or
        data.zip.zip, and cannot find data.zip.ZIP, period.
huang@DESKTOP-EV8L72E:/mnt/d/dev/groops/data$ ll data.zip
-rwxrwxrwx 1 huang huang 2248322672 Feb 27 12:32 data.zip*
huang@DESKTOP-EV8L72E:/mnt/d/dev/groops/data$
Regards, 
Huang

**Comment by tmayerguerr** [comment]
Hi Huang,

We are currently experiencing massive problems with our server, which is why the automatic GROOPS data service is not working properly. It will probably take a few more days until everything is working properly again.
I have uploaded the file data.zip once manually.

Best regards
Torsten

**Reply by tmayerguerr** [reply]
The servers should be working properly again.
